deployments/helm/KubeArmorOperator/README.md

```diff
@@ -66,6 +69,9 @@ spec:
     kubearmorRelayImage:
         image: [image-repo:tag]                                # DEFAULT - kubearmor/kubearmor-relay-server:latest
         imagePullPolicy: [image pull policy]                   # DEFAULT - Always
+        enableStdoutLogs: [show stdout logs]                   # DEFAULT - "false"
+        enableStdoutAlerts: [show stdout alerts]               # DEFAULT - "false"
+        enableStdoutMsgs: [show stdout messages]               # DEFAULT - "false" 
```

Comment:
@VedRatan I see we have kept these fields under `spec` at some places and under `kubearmorRelayImage` at others? Please keep one.
Since your initial implementation of the operator's code is still referring to them under `spec`, I think we can follow that.